__tests__/app.test.js

```diff
@@ -13,27 +13,59 @@ describe("/api", () => {
       .get("/api")
       .expect(200)
       .then(({ body }) => {
-        const regex = /(GET|POST|PATCH|DELETE) \/api(\S+)?\b/;
+        const regex = /^(GET|POST|PATCH|DELETE) \/api(\S+)?$/;
         for (let key in body) {
           expect(key).toMatch(regex);
-          expect(body[key]).toMatchObject({
-            description: expect.any(String),
-            queries: expect.any(Array),
-            exampleResponse: expect.any(Object)
-          });
+          if (key === "GET /api") {
+            expect(body[key]).toMatchObject({
+              description: expect.any(String)
+            });
+          } else {
+            expect(body[key]).toMatchObject({
+              description: expect.any(String),
+              queries: expect.any(Array),
+              exampleResponse: expect.any(Object)
```

Comment:
Awesome job and glad this method is working.
It would be perfectly acceptable to require in the endpoints.json file into your test and assert that your response looks the same.